test/fprime_gds/common/tools/seqgen_unit_test.py

```diff
@@ -1,7 +1,42 @@
+import filecmp
+from pathlib import Path
+import tempfile
 import unittest
 
+import fprime_gds.common.tools.seqgen as seqgen
 
 class APITestCases(unittest.TestCase):
     def test_nominal_sequence(self):
-
-        pass
+        self.assertEqual(self.diff_input_sequence(
+            "./input/simple_sequence.seq",
+            "./expected/simple_sequence_expected.bin",
+            "./resources/simple_dictionary.json"
+        ), True)
+    
+    # This test is expected to fail due to unmatched command in
+    # simple_bad_sequence.seq and seqgen should produce a SeqGenException. 
+    # Passing these executions is a test failure.
+    @unittest.expectedFailure 
```

Comment:
Instead of marking that you expect a test case to fail, I usually prefer using [`assertRaises`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises) to check that the method raises the expected assertion and isn't just generally crashing.